app/assets/javascripts/students/tests.coffee

```diff
@@ -0,0 +1,17 @@
+$ ->
+  liftOff = ->
+    $(window).unbind "beforeunload"
+
+  url = $(".test-form").attr("action")
+  time = $('#time_tab').data('content')
+  $('#time_tab').countdown({until: time, expiryUrl: url, onExpiry: liftOff, format: 'MS'})
+
+  if $('#time_section').length > 0
+    $(window).unload ->
+      url = $(".test-form").attr("action")
+      window.location = 'url'
```

Comment:
if user reload page form with test result will be sent to server and user will be redirected to page with result